Modules/nav-bar.js

```diff
@@ -0,0 +1,38 @@
+
+const allBooksSection = document.getElementById('all-books');
+const listLink = document.getElementById('list-link');
+const addLink = document.getElementById('add-link');
+const contactLink = document.getElementById('contact-link');
+const addNewBookSection = document.getElementById('add-new-book');
+const contactSection = document.getElementById('contact');
+
+allBooksSection.classList.add('active');
+allBooksSection.classList.remove('hide-class');
+
+function toggleVisbility(activeEle) {
+  activeEle.classList.toggle('active');
+  activeEle.classList.remove('hide-class');
+}
+
+function clearClasses(element1, element2) {
+  element1.classList.remove('active');
+  element1.classList.add('hide-class');
+
+  element2.classList.remove('active');
+  element2.classList.add('hide-class');
+}
```

Comment:
- [ ] Please as requested by the previous reviewer, it's one of the project requirements you use `arrow` syntax for functions instead of JavaScript Function Expressions, Kindly make sure to address the requested changes :wink: 💯 

- For more documentation, you can visit this [link](https://www.w3docs.com/learn-javascript/function-expressions.html)